screens/ChooseLocationScreen.tsx

```diff
@@ -95,9 +95,9 @@ export const ChooseLocationScreen = ({
             }
           }}
         >
-          {hotspots.map(h => (
+          {hotspots.map((h, index) => (
             <Marker
-              key={`${h.latitude} ${h.longitude}`}
+              key={index}
```

Comment:
Using the index as key is a bad practice in React. Is it possible to have 2 hotspots with the same coordinates?

Reply:
At this moment we have some, but it's created by me, so I'll but it back.

Reply:
Maybe we should have a check and prevent the user to add multiple hotspots in the same location and give a suggestion like: "that there already is a hotspot on that location - update that instead of adding a new one" 
We'll put it in the "backlog"